graveler/graveler_test.go

```diff
@@ -490,30 +504,14 @@ func TestGraveler_CommitExistingRange(t *testing.T) {
 			want:        expectedCommitID,
 			expectedErr: nil,
 		},
-		{
-			name: "staging is dirty",
```

Comment:
No sir, you can't. Mainly because you're committing an existing meta-range and simply link to some parent. We DON'T advance the branch ref here, so checking for dirty branch is irrelevant.